workflows/sc_adaptive/example_sc_adaptive.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+
+import boutvecma
+import easyvvuq as uq
+import chaospy
+import os
+import numpy as np
+import time
+import matplotlib
+# Do not open figures:
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+
+def refine_sampling_plan(number_of_refinements):
+    """
+    Refine the sampling plan.
+
+    Parameters
+    ----------
+    number_of_refinements (int)
+       The number of refinement iterations that must be performed.
+
+    Returns
+    -------
+    None. The new accepted indices are stored in analysis.l_norm and the admissible indices
+    in sampler.admissible_idx.
+    """
+    for i in range(number_of_refinements):
+        # compute the admissible indices
+        sampler.look_ahead(analysis.l_norm)
+
+        # run the ensemble
+###        campaign.draw_samples()
```

Comment:
Lots of commented out code in this example -- should be deleted?

Reply:
Thanks, I'll delete the obsolete code.  I'd like to keep some of the comments though, since they are somewhat in lieu of EasyVVUQ documentation...